care/facility/models/patient_icmr.py

```diff
@@ -45,26 +46,18 @@ class Meta:
     #         instance.__class__ = PatientSampleICMR
     #     return instance
 
+    def get_age_delta(self):
+        start = self.date_of_birth or timezone.datetime(self.year_of_birth, 1, 1).date()
+        end = (self.death_datetime or timezone.now()).date()
+        return relativedelta(end - start)
```

Comment:
dateutil.relativedelta package is only used here, can we remove it?
we could just convert dates to datetime with h:mm as 0:00
and return the resulting timedelta